doc/source/commands/files.rst

```diff
@@ -406,196 +415,10 @@ Example::
-will not actually occur if the program has already been run once during the
-current program context.  This is intended mostly for loading library program
-files that may have mainline code in them for initialization purposes that you
-don't want to get run a second time just because you use the library in two
-different subprograms.
-
-``RUN ONCE`` and ``RUNONCEPATH`` mean "Run unless it's already been run, in which
-case skip it."
-
-.. note::
-
-    *Limitations on file names used for programs*
-
-    On case-sensitive filesystems typically found on Linux and Mac, you should
-    name program files used with the ``run`` command entirely with
-    lowercase-only filenames or the system may fail to find them when you
-    use the ``run`` command.
-
-Arguments
-^^^^^^^^^
-
-Although the syntax is a bit different for ``RUN`` versus
-``RUNPATH`` (and ``RUNONCEPATH``), all 3 techniques allow you to
-pass arguments into the program that it sees as its main script
-:ref:`parameter <declare parameter>` values.
-
-The following commands do equivalent things::
-
-    RUN "AutoLaunch.ks"( 75000, true, "hello" ).
-    RUNPATH("AutoLaunch.ksm", 75000, true, "hello" ).
-
-In both of the above examples, had the program "AutoLaunch.ks"
-started with these lines::
-
-    // AutoLaunch.ks program file:
-    parameter final_alt, do_countdown, message.
-    //
-    // rest of program not shown...
-    //
-
-Then inside AutoLaunch.ks, ``final_alt`` would be ``75000``,
-and ``do_countdown`` would be ``true``, and ``message``
-would be ``"hello"``.
-
-Full path information is allowed
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-You are allowed to use directory path information
-in the string filename you are running (even with
-the older ``RUN`` command)::
-
-    // Absolute path names are allowed.
-    RUN "0:/archive_lib/myfile.ks".
-    RUN ONCE "0:/archive_lib/myfile.ks".
-    RUNPATH("0:/archive_lib/myfile.ks").
-    RUNONCEPATH("0:/archive_lib/myfile.ks").
-
-    // Relative path names are also allowed.
-    RUN "../dir1/myfile.ks"(arg1, arg2).
-    RUNPATH("../dir1/myfile.ks", arg1, arg2).
-
-Automatic guessing of full filename
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-For all 3 types of run command (``RUN``, ``RUNPATH``, and ``RUNONCEPATH``),
-the following filename "guess" rules are used when the filename given is
-incomplete:
-
-- 1: If no path information was present in the filename, then assume the
-  file is in the current directory (that's pretty much standard for all
-  filename commands).
-
-- 2: Assume if no filename extension such as ".ks" or ".ksm" was given,
-  and there is no file found that lacks an extension in the way the
-  filename was given, then first try to find a file with the ".ksm"
-  extension appended to it, and if that file is not found then try
-  to find a file with the ".ks" extension appended to it.
+You may run saved script files using the various :ref:`Run Command<running>`.
```

Comment:
I don't mind adding the basic example and that closing statement.  That's short enough and not detailed enough to cause issues with maintenance.

But I would really prefer not to provide any qualified statements like talking about file names or contents.  I like those details being on the dedicated page.  That way they're all in one place if we change them in the future.  It isn't a big enough detail that I'll argue about it, but thought I should at lest mention it.  It may be a good place for a `.. seealso::` clause to call special attention to the link.